app/models/runtime/route.rb

```diff
@@ -183,6 +183,10 @@ def tcp?
       domain.shared? && domain.tcp? && port.present? && port > 0
     end
 
+    def protocol
+      self.domain.protocols.first
```

Comment:
How does the route's protocol interplay with a destination's protocol?  It might be helpful to clarify the new behavior in the [Route Object's documentation](https://v3-apidocs.cloudfoundry.org/version/3.102.0/index.html#the-route-object) since the description seems to be out of date: 

```
The protocol supported by the route, based on the route’s domain configuration. Valid protocols are http and tcp
```

Reply:
That description looks correct to me. We expanded upon the connection between route and route destination protocols in the route destination documentation instead of in the route documentation.

Reply:
Makes sense.  Before our sync last week, I was expecting `domain.protocol` to support `tcp`, `http1`, or `http2`, but understand now that it is only on the route destination where the distinction between `http1` and `http2` is made.